core/auth/authGroup.js

```diff
@@ -46,6 +46,66 @@ export const groupNames = {
   surveyGuest: 'surveyGuest',
 }
 
+const permissionsByGroupName = {
```

Comment:
I think that we have this list in another part of the app, or in arena-core

Reply:
updated, the other part of the code uses this one